homeassistant/components/uvc/camera.py

```diff
@@ -119,6 +119,11 @@ def motion_detection_enabled(self):
         """Camera Motion Detection Status."""
         return self._caminfo["recordingSettings"]["motionRecordEnabled"]
 
+    @property
+    def unique_id(self) -> str:
+        """Return a unique identifier for this client."""
+        return self._caminfo["uuid"]
```

Comment:
According to and comment on line 48, this property is not available all the time.
Maybe instead of the identifier determined at line 48?

```suggestion
        return self._uuid
```

Reply:
Swapped to use the identifier.